packages/apps/human-app/server/src/interfaces/signup-worker-request.dto.ts

```diff
@@ -1,5 +1,9 @@
+import { ApiHideProperty, ApiProperty } from '@nestjs/swagger';
 import { IsEmail, IsString } from 'class-validator';
-import { ApiProperty } from '@nestjs/swagger';
+
+export enum WorkerType {
+  WORKER = 'WORKER',
+}
 
 export class SignupWorkerDto {
```

Comment:
Please create additional `SignupWorkerCommand` to use in the business layer.

This DTO object should have just `email` and `password` fields

Reply:
Fixed in commit 8300eb3f